pandas/tests/frame/test_sorting.py

```diff
@@ -496,12 +496,22 @@ def test_sort_index_duplicates(self):
     def test_sort_index_level(self):
         mi = MultiIndex.from_tuples([[1, 1, 3], [1, 1, 1]], names=list('ABC'))
         df = DataFrame([[1, 2], [3, 4]], mi)
+
         res = df.sort_index(level='A', sort_remaining=False)
         assert_frame_equal(df, res)
 
         res = df.sort_index(level=['A', 'B'], sort_remaining=False)
         assert_frame_equal(df, res)
 
+        res = df.sort_index(level=['C', 'B', 'A'])
+        assert_frame_equal(df.iloc[[1, 0]], res)
```

Comment:
write these as

expected = ....
assert_frame_equal(result, expected)

need also to fix the others in this test